classes/fields/pick.php

```diff
@@ -2939,12 +2940,18 @@ public function admin_ajax_relationship() {
 		}
 
 		$pod   = self::$api->load_pod( array( 'id' => (int) $params->pod ) );
-		$field = self::$api->load_field(
-			array(
-				'id'         => (int) $params->field,
-				'table_info' => true,
-			)
-		);
+
+		$field = Store::get_instance()->get_object( $params->field );
```

Comment:
Unfortunately this store call is not the correct fix even though it solves it for your specific case. Store calls are all limited across Pods to PodsAPI so that it can properly handle multiple stores and prevent conflicts with fields from other pods too.

The correct fix would be to adjust the code so that it's not looking up by ID but by the field name while also passing the pod name. Pods is still transitioning from reliance on IDs towards using names/slugs in more places for better support for code/file based configs like this.

There may be other areas of the code that need to be adjusted further too in this way, but the last big area is definitely sorting out the wp_podsrel table and writing a migration script that can deal with lots of data.

That's all a big undertaking and not part of this fix but I wanted to give you the big picture of what we're doing and where we're going with things like this.

I'll push some updates after I get some fixes in, but I'm also going to write a test for this method as well to make sure it's totally solved and won't break again.